plugins/auth-backend/src/service/router.ts

```diff
@@ -158,3 +161,18 @@ export async function createRouter({
 
   return router;
 }
+
+function createOriginFilter(config: Config): (origin: string) => boolean {
+  const allowedOrigins = config.getOptionalStringArray('auth.allowedOrigins');
```

Comment:
Let's make this `auth.experimentalExtraAllowedOrigins` for now, and skip adding it to config schema as well